frontend/src/components/ImageComponent.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent } from 'react';
+
+import styled from 'styled-components';
+
+import { API_SERVER } from '../data/ApiConstants';
+import Image from '../models/Image';
+
+const BorderedImage = styled.img`
+  height: 100%;
+  width: 100%;
+  border: 1px solid #ddd;
+  border-radius: 0.15em;
+  padding: 0.15em;
+`;
```

Comment:
This one _could_ be converted to the `antd` equivalent. Wanted to know what the convention is for this custom-ish stuff. Similarly for the styled caption below, I really have no idea the best way to tidy this up i.e. as is versus

```javascript
import { Image } from 'antd';
...
<figure>
    <Image src={API_SERVER + image.href} width="100%" height="100%" alt={image.title} />
    <CenteredCaption>{image.title}</CenteredCaption>
</figure>
```

Reply:
Yeah this could probably be a `Card` from Ant ([docs](https://ant.design/components/card/#components-card-demo-flexible-content)). There's no real convention for this kind of stuff but images with captions are often just stuffed into cards. The `figure` and `figcaption` elements are more for the article context (bit like LaTeX in that regard) but there's no hard and fast rule for that